### PR TITLE
clean up some waffle switches

### PR DESCRIPTION
**Pull request by andymckay**
Some quick clean up wins:

* allow-refund: never got deleted properly, not mentioned in code
* paypal-disable: never got deleted properly, not mentioned in code
* browserid-login: wat? not mentioned in code
* video-encode: created in 2012, never turned on, is this ever going to be turned on? no, it's dead.
* collection-stats: created in 2012, not turned on
* theme-stats: created in 2013, not turned on

### CODE REVIEW

File: lib/settings_base.py [hunk 1]
```diff
@@ -471,7 +471,6 @@ def JINJA_CONFIG():
             'css/impala/moz-tab.css',
             'css/impala/footer.less',
             'css/impala/faux-zamboni.less',
-            'css/impala/collection-stats.less',
```
Comment (magopian) [inline lib/settings_base.py]: wow, good catch ;)

File: lib/video/tasks.py [hunk 2]
```diff
@@ -85,12 +71,7 @@ def _resize_video(src, instance, **kw):
             os.makedirs(dirs)
 
     shutil.move(thumbnail_file, instance.thumbnail_path)
-    if waffle.switch_is_active('video-encode'):
-        # Move the file over, removing the temp file.
-        shutil.move(video_file, instance.image_path)
-    else:
-        # We didn't re-encode the file.
-        shutil.copyfile(src, instance.image_path)
```
Comment (magopian) [inline lib/video/tasks.py]: Don't we want to keep that?

Comment (magopian) [inline lib/video/tasks.py]: well, i'm not reading today. Forget about that previous comment ;)

**Comment by magopian** [comment]
Two minor flake8 warnings about unused `waffle` imports.

Apart from that, all good, thanks for the cleanup!
r+wc

**Comment by andymckay** [comment]
It would help if I included the database migration, which I didn't. Will do that when I get to the office tomorrow.

**Comment by kumar303** [comment]
I think some of these might be Marketplace ghosts

**Comment by andymckay** [comment]
What does that mean?

File: lib/video/tasks.py [hunk 3]
```diff
@@ -85,12 +71,7 @@ def _resize_video(src, instance, **kw):
             os.makedirs(dirs)
 
     shutil.move(thumbnail_file, instance.thumbnail_path)
-    if waffle.switch_is_active('video-encode'):
-        # Move the file over, removing the temp file.
-        shutil.move(video_file, instance.image_path)
-    else:
-        # We didn't re-encode the file.
```
Comment (kumar303) [inline lib/video/tasks.py]: should this comment be kept? I'm not sure exactly what it means.

File: static/css/impala/collection-stats.less [hunk 4]
```diff
@@ -1,13 +0,0 @@
-@import 'lib';
-
-.widgets {
-    .stats {
-        background: url("../../img/impala/stats.png") no-repeat left top;
```
Comment (kumar303) [inline static/css/impala/collection-stats.less]: is `stats.png` still needed?

File: lib/video/tasks.py (hunk 3 above)
Comment (andymckay) [inline lib/video/tasks.py]: it means the if statement didn't happen, i'm not sure it was helpful the first time around

File: static/css/impala/collection-stats.less (hunk 4 above)
Comment (andymckay) [inline static/css/impala/collection-stats.less]: yeah its used in other places

**Comment by kumar303** [comment]
> What does that mean?

Nothing, just me thinking out loud. I thought maybe the video encode was a marketplace thing that didn't get removed in the zamboni cut-over. Maybe not though.

**Comment by kumar303** [comment]
r+wc

**Pull request merged**